linux-tkg/linux-tkg-config/prepare

```diff
@@ -0,0 +1,1262 @@
+#!/bin/bash
+
+_tkg_initscript() {
+
+  # create build dir early
+  if [ "$_distro" = "Void" ]; then
+    _path="${XBPS_BUILDDIR}/${wrksrc}"
+    if [ ! -e ${_path} ]; then
+      mkdir ${_path}
+    fi
+  else
+    _path="${_where}"
+  fi
+
+  if [ -z "$_version" ] && [ ! -e "$_path"/versel ]; then
+    plain "Which kernel version do you want to install?"
+    read -rp "`echo $'  > 1. 5.8\n    2. 5.9-rc\n    3. 5.7\n    4. 5.4\nchoice[1-4?]'`" _VERSEL;
+    case $_VERSEL in
+          "2")
+          echo "_basever=59" > "$_path"/versel
+          echo "_basekernel=5.9" >> "$_path"/versel
+          echo "_sub=rc4" >> "$_path"/versel
+          ;;
+          "3")
+          echo "_basever=57" > "$_path"/versel
+          echo "_basekernel=5.7" >> "$_path"/versel
+          echo "_sub=19" >> "$_path"/versel
+          ;;
+          "4")
+	  echo "_basever=54" > "$_path"/versel
+	  echo "_basekernel=5.4" >> "$_path"/versel
+	  echo "_sub=64" >> "$_path"/versel
+	  ;;
+          *)
+          echo "_basever=58" > "$_path"/versel
+          echo "_basekernel=5.8" >> "$_path"/versel
+          echo "_sub=8" >> "$_path"/versel
+          ;;
+    esac
+  elif [ -n "$_version" ];then
+    case "$_version" in
+		"5.4")
+		echo "_basever=54" > "$_path"/versel
+		echo "_basekernel=5.4" >> "$_path"/versel
+		echo "_sub=64" >> "$_path"/versel
+		;;
+		"5.7")
+		echo "_basever=57" > "$_path"/versel
+		echo "_basekernel=5.7" >> "$_path"/versel
+		echo "_sub=19" >> "$_path"/versel
+		;;
+		"5.8")
+		echo "_basever=58" > "$_path"/versel
+		echo "_basekernel=5.8" >> "$_path"/versel
+		echo "_sub=8" >> "$_path"/versel
+		;;
+		"5.9")
+		echo "_basever=59" > "$_path"/versel
+		echo "_basekernel=5.9" >> "$_path"/versel
+		echo "_sub=rc4" >> "$_path"/versel
+		;;
+		*)
+		error "There is something wrong with your kernel version selection, exiting..."
+		exit 1
+	esac
+  fi
+  if [ "$_distro" != "Void" ]; then
+    cp "$_where"/linux-tkg-patches/* "$_where" # copy patches inside the PKGBUILD's dir to preserve makepkg sourcing and md5sum checking
+    cp "$_where"/linux-tkg-config/* "$_where" # copy config files and hooks inside the PKGBUILD's dir to preserve makepkg sourcing and md5sum checking
+  else
+    cp "$_where"/linux-tkg-patches/* "$_path"
+    cp "$_where"/linux-tkg-config/* "$_path"
+  fi
+
+  # Load external configuration file if present. Available variable values will overwrite customization.cfg ones.
```

Comment:
This should be moved to the start of `_tkg_initscript` as otherwise `_distro` and `_version` will not be loaded from external configuration which is needed before the current place that loading of external configuration file occurs